.github/workflows/linter.yml

```diff
@@ -13,9 +13,10 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v2
       - name: Lint Code Base
-        uses: docker://github/super-linter:v3.12.0
+        uses: docker://github/super-linter:v4.5.0
         env:
           VALIDATE_ALL_CODEBASE: false
+          VALIDATE_JSCPD: false
```

Comment:
If enabled it will complain about duplication. For ingress I am not sure if it's worth refactoring it.

```
Clone found (yaml):
 - /tmp/lint/charts/jenkins/templates/jenkins-controller-ingress.yaml [60:7 - 71:5] (11 lines, 113 tokens)
   /tmp/lint/charts/jenkins/templates/jenkins-controller-ingress.yaml [37:7 - 48:2]

Clone found (yaml):
 - /tmp/lint/charts/jenkins/templates/jenkins-controller-ingress.yaml [60:7 - 71:5] (11 lines, 113 tokens)
   /tmp/lint/charts/jenkins/templates/jenkins-controller-ingress.yaml [37:7 - 48:2]

 60 │ 37 │ - backend:                                                  
 61 │ 38 │ {{- if semverCompare ">=1.19-0" $kubeTargetVersion }}       
 62 │ 39 │           service:                                          
 63 │ 40 │             name: {{ template "jenkins.fullname" . }}       
 64 │ 41 │             port:                                           
 65 │ 42 │               number: {{ .Values.controller.servicePort }}  
 66 │ 43 │         pathType: ImplementationSpecific                    
 67 │ 44 │ {{- else }}                                                 
 68 │ 45 │           serviceName: {{ template "jenkins.fullname" . }}  
 69 │ 46 │           servicePort: {{ .Values.controller.servicePort }} 
 70 │ 47 │ {{- end }}                                                  
 71 │ 48 │     host                                                    

Found 1 clones.

```